cli/integration_tests/dry_json/monorepo.t

```diff
@@ -120,13 +121,16 @@ Setup
     },
     "expandedOutputs": [],
     "framework": "<NO FRAMEWORK DETECTED>",
+    "envMode": "infer",
     "environmentVariables": {
       "configured": [],
       "inferred": [],
       "global": [
         "SOME_ENV_VAR=",
         "VERCEL_ANALYTICS_ID="
-      ]
+      ],
+      "passthrough": null,
+      "globalPassthrough": null
```

Comment:
not to self, this is null because it's not included in the config. if the config sets it to an empty array that means something. so this is good 👍🏾